toolchest_client/tools/api.py

```diff
@@ -388,6 +391,44 @@ def megahit(output_path=None, tool_args="", read_one=None, read_two=None, interl
     return output
 
 
+def python3(script, inputs=[], output_path=None, tool_args="", **kwargs):
+    """Runs Python3 via Toolchest.
+    NOTE: This tool is currently only available for customers with contracts
+    The script has a few requirements if inputs or outputs are expected. Inputs will be uploaded to `./input/` so the
+    script must use that prefix with hardcoded paths or when adding to tool_args to be parsed via sys.argv. Any output
+    that is expected to be returned needs to be stored at `./output/`. The directory will be compressed via pigz and
+    decompressed on the client.
```

Comment:
This can be rephrased to be more digestible

Reply:
"The script has a few requirements if inputs or outputs are expected." – not necessary

"Inputs will be uploaded to `./input/` so the script must use that prefix with hardcoded paths or when adding to tool_args to be parsed via sys.argv" - hard to understand

"Any output that is expected to be returned needs to be stored at `./output/`" - make more clear what this means, what needs to store what at ./output/?

"The directory will be compressed via pigz and decompressed on the client." - more detail than is necessary, but still lacks implication (maybe "The output directory is compressed and returned, meaning chaining Python 3 calls via Toolchest is not possible")